Source/Dafny/Compiler-go.cs

```diff
@@ -3312,8 +3312,6 @@ private class SpecialNativeType : UserDefinedType {
     protected override string GetCollectionBuilder_Build(CollectionType ct, Bpl.IToken tok, string collName, TargetWriter wr) {
       if (ct is SetType) {
         return collName + ".ToSet()";
-      } else if (ct is MultiSetType) {
-        return collName + ".ToMultiSet()";
```

Comment:
Why is it safe to remove this?

Reply:
There is no multiset comprehension in Dafny.